apps/envited-x-data-space/common/serverActions/globalIdentifier/globalIdentifier.ts

```diff
@@ -0,0 +1,37 @@
+import { isEmpty, isNil } from 'ramda'
+
+import { getServerSession } from '../../auth'
+import { db } from '../../database/queries'
+import { Database } from '../../database/types'
+import { parseGlobalIdentifier } from '../../globalIdentifiers'
+import { Log, log } from '../../logger'
+import { Session } from '../../types'
+import { badRequestError, formatError, internalServerErrorError, unauthorizedError } from '../../utils'
+
+export const _getGlobalIdentifierById =
```

Comment:
I'm not sure what youre doing here, you're giving it the full uri syntax it seems like, to return just about the same thing?

If you say ById, I'd assume you are giving it the db uuid, not the full uri.